astroquery/gaia/tests/test_gaiatap.py

```diff
@@ -16,6 +16,8 @@
 """
 import unittest
 import os
+import pytest
```

Comment:
When I ported Cone Search over, I saw that some of your modules still import `pytest` from Astropy, so I did as well. If you want to use pytest that is not bundled with Astropy here, all the other modules should do the same.